unix/build_dist.sh

```diff
@@ -64,9 +64,12 @@ done
 cp "$ALS_PROLOG/LICENSE.txt" "$DISTDIR/LICENSE.txt"
 cp -p "$MAN/welcome_standard.txt" "$DISTDIR/README.txt"
 cp -p $MAN/$MANUAL "$DISTDIR/$MANUALNAME"
-mkdir "$DISTDIR/help"
-cp -pr $MAN/$HELP/* "$DISTDIR/help"
-cp -p $MAN/als_help.htm "$DISTDIR/als_help.htm"
+mkdir "$DISTDIR/alshelp"
+cp -pr $MAN/$HELP/* "$DISTDIR/alshelp"
+cp -p $MAN/als_help.html "$DISTDIR/als_help.html"
+cp -p $MAN/alshelp.css "$DISTDIR/alshelp.css"
+cp -p $MAN/package_nav.html "$DISTDIR/package_nav.html"
+
```

Comment:
Bah, I was thinking of it & forgot.

On Dec 9, 2015, at 6:02 AM, Chuck Houpt <notifications@github.com> wrote:

> In unix/build_dist.sh:
> 
> > @@ -64,9 +64,12 @@ done
> >  cp "$ALS_PROLOG/LICENSE.txt" "$DISTDIR/LICENSE.txt"
> >  cp -p "$MAN/welcome_standard.txt" "$DISTDIR/README.txt"
> >  cp -p $MAN/$MANUAL "$DISTDIR/$MANUALNAME"
> > -mkdir "$DISTDIR/help"
> > -cp -pr $MAN/$HELP/* "$DISTDIR/help"
> > -cp -p $MAN/als_help.htm "$DISTDIR/als_help.htm"
> > +mkdir "$DISTDIR/alshelp"
> > +cp -pr $MAN/$HELP/* "$DISTDIR/alshelp"
> > +cp -p $MAN/als_help.html "$DISTDIR/als_help.html"
> > +cp -p $MAN/alshelp.css "$DISTDIR/alshelp.css"
> > +cp -p $MAN/package_nav.html "$DISTDIR/package_nav.html"
> > +
> 
> Might also want to include the reference PDF, which was recently added.
> 
> �
> Reply to this email directly or view it on GitHub.
>